tests/jest/components/CreateApplicationPage/controlData/ControlData.test.js

```diff
@@ -1041,7 +1041,7 @@ describe("reverseExistingRule with template data", () => {
     active: "-placement-1",
     groupControlData: [
       {
-        active: "-placement-1",
+        // active: "-placement-1",
```

Comment:
If we no longer need the active in groupControlData we might want to delete it? I'm fine with it being a comment so will approve